libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/router/RouterInterfaceAdapter.kt

```diff
@@ -142,7 +142,8 @@ class RouterInterfaceAdapter(
                     )
 
                 ),
-                com.mapbox.navigator.RouterOrigin.CUSTOM
+                com.mapbox.navigator.RouterOrigin.CUSTOM,
+                hashMapOf()
```

Comment:
what `hashMap` should we pass here and below?

Reply:
There are refresh adapters and we don't pass the header from native further anyway, so empty maps are fine for now.